contents/teams/mobile/objectives.mdx

```diff
@@ -1,7 +1,36 @@
-### Q4 2025 Objectives
+### Q1 2026 Objectives
 
-#### Ship mobile error tracking (Driver: <TeamMember name="Manoel Aranda Neto" />)
-* Complete 'straightforward' section of the [megaissue](https://github.com/PostHog/posthog/issues/38845)
-* Support suppression rules
-* Add suppression of minified exceptions before cymbal processing
-* Build dashboard template including crash free sessions / users metric
+#### Goal 1: Replay cost control - [parity tracker issue](https://github.com/PostHog/posthog/issues/30889) - (Driver: <TeamMember name="Ioannis Josephides" />)
+
+* [Programmatically start and stop recordings](https://posthog.com/docs/session-replay/how-to-control-which-sessions-you-record#programmatically-start-and-stop-recordings) (React Native and Flutter)
+* [Min. recording duration](https://posthog.com/docs/session-replay/how-to-control-which-sessions-you-record#minimum-duration) (SDKs TBD)
+* [Sampling rate](https://posthog.com/docs/session-replay/how-to-control-which-sessions-you-record#sampling) (SDKs TBD)
+* Event triggers (SDKs TBD)
```

Comment:
I'm doing a lot of work on this right now. I'm abstracting the replay ingestion controls because we want to reuse them for error tracking too. I'm making it configurable so that different settings can be applied to different contexts (product, library, etc). Obviously this will work like replay where the config is sent down remotely - you will still need to do the SDK implementation yourselves.